### ISSUE TITLE
 RayTraceBenchmark/Cpp/Results.md is empty

### ISSUE THREAD

**Issue by underscoreanuj** [zezba9000/RayTraceBenchmark#3]
https://github.com/zezba9000/RayTraceBenchmark/blob/master/Cpp/Results.md

**Comment by zezba9000** [comment]
Yep for some reason when these were done they were never pushed up.
You can just run the benchmark on a computer you want to compare the results.
